Bio/PDB/MMCIFParser.py

```diff
@@ -229,9 +229,7 @@ def _build_structure(self, structure_id):
                 # Non-existing residue ID
                 try:
                     msg_resseq = mmcif_dict["_atom_site.auth_seq_id"][i]
-                    msg = "Non-existing residue ID in chain '{}', residue '{}'".format(
```

Comment:
These changes are amazing. I didn't know it would automagically do this!

Reply:
I was impressed too :)